airflow/www/views.py

```diff
@@ -1265,6 +1283,7 @@ def set_duration(tid):
 
         form = DateTimeWithNumRunsForm(data={'base_date': max_date,
                                              'num_runs': num_runs})
+        external_logs = conf.get('elasticsearch', 'elasticsearch_frontend')
```

Comment:
For future reference this should be

```suggestion
        external_logs = conf.getboolean('elasticsearch', 'elasticsearch_frontend')
```